mlflow/gateway/schemas/completions.py

```diff
@@ -1,8 +1,17 @@
 from typing import Dict, List, Optional
 
-from mlflow.gateway.base_models import ResponseModel
+from pydantic import Field
+
+from mlflow.gateway.base_models import RequestModel, ResponseModel
 from mlflow.gateway.config import RouteType
-from mlflow.gateway.schemas.chat import BaseRequestPayload, FinishReason
+from mlflow.gateway.schemas.chat import BaseRequestPayload
+
+
+class BaseRequestPayload(RequestModel):
+    temperature: float = Field(0.0, ge=0, le=2)
+    n: int = Field(1, ge=1, le=5)
+    stop: Optional[List[str]] = Field(None, min_items=1)
+    max_tokens: Optional[int] = Field(None, ge=1)
```

Comment:
Copy the previous `BaseRequestPayload` from `chat.py` into `completions.py` so we can just make chat-related changes in this PR. Completions will be a follow-up PR